content/develop/connect/redis-for-vscode.md

```diff
@@ -0,0 +1,124 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Connect to Redis from Visual Studio Code.
+linkTitle: Redis for VS Code
+stack: true
+title: Redis for VS Code
+weight: 5
+---
+
+Redis for VS Code is an extension that allows you to connect to your Redis databases from within Microsoft Visual Studio Code.
+After connecting to a database, you can view, add, modify, and delete keys, and interact with your Redis databases using a built-in CLI interface.
+The following data types are supported:
+
+- [Hash]({{< relref "/develop/data-types/hashes" >}})
+- [List]({{< relref "/develop/data-types/lists" >}})
+- [Set]({{< relref "/develop/data-types/sets" >}})
+- [Sorted Set]({{< relref "/develop/data-types/sorted-sets" >}})
+- [String]({{< relref "/develop/data-types/strings" >}})
+- [JSON]({{< relref "/develop/data-types/json" >}})
+
+## Install the Redis for VS Code extension
+
+Open VS Code and click on the **Extensions** menu button. In the **Search Extensions in Marketplace** field, type "Redis for VS Code" and press the `enter` or `return` key. There may be more than one option shown, so be sure to click on the extension published by Redis.
+
+**[DWD] flesh out with more instructions once the extension is available in the marketplace. [DWD]**
+
+After installing the extension, your VS Code menu will look similar to the following.
+
+{{< image filename="images/dev/connect/vscode/vscode-menu.png" >}}
+
+The Redis for VS Code extension will automatically update just like any other VS Code extension.
+
+## Connect to Redis databases {#connect-db}
+
+Click on the Redis mark (the cursive **R**) in the VS Code menu to begin connecting a Redis database to VS Code.
+
+{{< image filename="images/dev/connect/vscode/vscode-initial.png" >}}
+
+Click on the **+ Connect database** button. A dialog will display in the main pane. In the image shown below, all the options have been checked to show the available details for each connection. These connection details are similar to those accessible from [`redis-cli`]({{< relref "/develop/connect/cli" >}}).
+
+{{< note >}}
+There is no way to change the logical database once you have selected it. If you need to connect to a different logical database, you need to add a separate database connection.
+{{< /note >}}
+
+{{< image filename="images/dev/connect/vscode/vscode-add-menu.png" >}}
+
+After filling out the necessary fields, click on the **Add Redis database** button. The pane on the left side, where you would normally see the Explorer view, shows your database connections.
```

Comment:
let's add a note that if they do not have a Redis db, they can create a free Cloud db or use Redis on Docker?

Reply:
This is made plain in the VS Code interface itself. No reason to document it again.